src/Fetch.fs

```diff
@@ -379,9 +379,23 @@ let postRecord<'T> (url: string) (record:'T) (properties: RequestProperties list
     // override the default ones if necessary
     List.append defaultProps properties
     |> fetch url
+    
 let tryPostRecord<'T> (url: string) (record:'T) (properties: RequestProperties list) : JS.Promise<Result<Response, Exception>> =
     postRecord url record properties |> Promise.result
 
+// Sends a HTTP put with the record serialized as JSON.
+// This function already sets the HTTP Method to PUT, sets the json into the body.

```

Comment:
Herp derp. Not sure how I missed that when reviewing the diff. Thanks for the catch, fixed!